resources/api/openapi.yml

```
@@ -201,54 +233,46 @@ components:
         ends_at:
           $ref: '#/components/schemas/DateTime'
         location:
-          $ref: '#/components/schemas/Location'
+          $ref: '#/components/schemas/Reference'
         shift_type:
-          $ref: '#/components/schemas/ShiftType'
+          $ref: '#/components/schemas/Reference'
         created_at:
           $ref: '#/components/schemas/DateTimeOptional'
         updated_at:
           $ref: '#/components/schemas/DateTimeOptional'
-        entries:
+        angel_types:
```

```suggestion
        needed_angel_types:
```
angel_types is confusing to me at this point
if you change it, it needs to be changed everywhere